src/backends/ilp-quoter/index.js

```diff
@@ -30,9 +30,13 @@ class ILPQuoter {
 
   * putPair (uri) {
     const req = request({ method: 'PUT', uri, json: true })
-    const res = yield req
-    if (res.statusCode >= 400) {
-      throw new UnsupportedPairError(req.uri)
+    try {
+      const res = yield req
+      if (res.statusCode >= 400) {
```

Comment:
In fact, it's probably best to check just for the exact error code the quoter is supposed to generate for this case and pass the error on for all other cases